tests/ml_models/test_mcorr.py

```diff
@@ -0,0 +1,179 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we create a folder (`metrics_correlation` may be?) under `tests/ml_models` and then put all the mcorr related test files inside that folder?